plugins/cypher-language/src/main/java/org/polypheny/db/cypher/clause/CypherCreateConstraint.java

```diff
@@ -74,12 +74,12 @@ public CypherCreateConstraint(
 
 
     @Override
-    public void execute( Context context, Statement statement, QueryParameters parameters ) {
+    public void execute( Context context, Statement statement, ParsedQueryContext parsedQueryContext ) {
         if ( constraintType != ConstraintType.UNIQUE ) {
             throw new UnsupportedOperationException( "Only unique constraints are supported at the moment." );
         }
 
-        //DdlManager.getInstance().addConstraint(  );
+        //DdlManager.getInstance().createConstraint(  );
```

Comment:
This statement should create a constraint, but it silently doesn't...

Reply:
Replaced with an explicit error.